### PR TITLE
Add missing bind* tests to object-deletion-behaviour.html

### PR DESCRIPTION
**Pull request by Jiawei-Shao**
Add cases on bindBufferBase and bindBufferRange to object-deletion-behaviour.html

### CODE REVIEW

**Comment by Jiawei-Shao** [comment]
@qkmiao  Please help to review

**Comment by qkmiao** [comment]
LGTM

File: sdk/tests/conformance/misc/object-deletion-behaviour.html [hunk 1]
```diff
@@ -376,6 +376,48 @@
...
+  shouldBe("gl.getParameter(gl.TRANSFORM_FEEDBACK_BUFFER_BINDING)", "bufferBaseTransformFeedback");
+  shouldGenerateGLError(gl, gl.NO_ERROR, "gl.deleteBuffer(bufferBaseTransformFeedback)");
+  shouldBeFalse("gl.isBuffer(bufferBaseTransformFeedback)");
+  shouldBeNull("gl.getParameter(gl.TRANSFORM_FEEDBACK_BUFFER_BINDING)");
+  shouldGenerateGLError(gl, gl.INVALID_OPERATION, "gl.bindBufferBase(gl.TRANSFORM_FEEDBACK_BUFFER, 0, bufferBaseTransformFeedback)");
+  shouldBeNull("gl.getParameter(gl.TRANSFORM_FEEDBACK_BUFFER_BINDING)");
+
+  var bufferRangeUniform = gl.createBuffer();
+  shouldBeNonNull("bufferRangeUniform");
+  shouldGenerateGLError(gl, gl.NO_ERROR, "gl.bindBufferRange(gl.UNIFORM_BUFFER, 0, bufferRangeUniform, 0, 4)");
```
Comment (zhenyao) [inline sdk/tests/conformance/misc/object-deletion-behaviour.html]: offset == 4 may not always work.  It needs to be a multiple of UNIFORM_BUFFER_OFFSET_ALIGNMENT

File: sdk/tests/conformance/misc/object-deletion-behaviour.html [hunk 2]
```diff
@@ -376,6 +376,48 @@
...
+  shouldBe("gl.getParameter(gl.UNIFORM_BUFFER_BINDING)", "bufferRangeUniform");
+  shouldGenerateGLError(gl, gl.NO_ERROR, "gl.deleteBuffer(bufferRangeUniform)");
+  shouldBeFalse("gl.isBuffer(bufferRangeUniform)");
+  shouldBeNull("gl.getParameter(gl.UNIFORM_BUFFER_BINDING)");
+  shouldGenerateGLError(gl, gl.INVALID_OPERATION, "gl.bindBufferRange(gl.UNIFORM_BUFFER, 0, bufferRangeUniform, 0, 4)");
```
Comment (zhenyao) [inline sdk/tests/conformance/misc/object-deletion-behaviour.html]: Same here, 4 is problematic.

File: sdk/tests/conformance/misc/object-deletion-behaviour.html [hunk 3]
```diff
@@ -376,6 +376,48 @@
 shouldGenerateGLError(gl, gl.INVALID_OPERATION, "gl.bindBuffer(gl.ELEMENT_ARRAY_BUFFER, bufferElement)");
 shouldBeNull("gl.getParameter(gl.ELEMENT_ARRAY_BUFFER_BINDING)");
 
+if (contextVersion > 1) {
```
Comment (kenrussell) [inline sdk/tests/conformance/misc/object-deletion-behaviour.html]: Could you please create a new conformance2/misc/object-deletion-behaviour.html and put the tests of these new object types in there? In general we have adhered to the policy of not putting WebGL 2.0-specific tests into conformance/, but rather into conformance2/. Thanks.

**Comment by kenrussell** [comment]
Thanks for revising and reorganizing the test. Looks good.

**Pull request merged**